src/mlpack/methods/ann/layer/isru.hpp

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file isru.hpp
+ * @author Prince Gupta
+ *
+ * Definition of ISRU (Inverse Square Root Unit) activation function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_ISRU_HPP
+#define MLPACK_METHODS_ANN_LAYER_ISRU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artifical Neural Network. */ {
+
+/**
+ * The ISRU (Inverse Square Root Unit) activation function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) &=& x / sqrt(1 + alpha* x ^ {2})
+ * f'(x) &=& cube(1 / sqrt(1 + alpha * x ^ {2}))
+ * @f
+ */
+template <
+    typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat
+>
+class ISRU
+{
+ public:
+  /**
+   * Create ISRU object using specified hyperparameter alpha.
+   * Default (alpha = 0.1). Alpha should be > 0.
+   * ISRU is defined as f(x) = x / (1 + alpha * x^{2})^{1/2}.
+   * 
+   * @param alpha hyperparameter used to calculate ISRU function.
```

Comment:
`hyperparameter` should start with a capital letter. What about we are more descriptive about `alpha`, something like: Hyperparameter alpha controls the value to which an ISRLU saturates for negative inputs (default = 0.1).